lib/render.js

```diff
@@ -79,7 +83,11 @@ var render = module.exports = function (html, options) {
   if (options.highlightSyntax) parser.use(codeWrap)
   if (options.serveImagesWithCDN) parser.use(cdnImages, {package: options.package})
 
-  return githubLinkify(parser).render(html)
+  return githubLinkify(parser)
+}
+
+render.renderPackageDescription = function (description) {
```

Comment:
That's the name this method has had since before I started working on marky; last I saw many months ago, the npm web site was using it for rendering package.json `description` fields into HTML for use in on package pages.

If they're not using it anymore, I'd be in favor of deprecating/removing it. People will be able to do basically the same thing by calling `marky.getParser().renderInline(markdownSnippet)`, but honestly it wouldn't surprise me if nobody at all is using it anyway.